nailgun/entities.py

```diff
@@ -4036,6 +4037,70 @@ def errata(self, synchronous=True, **kwargs):
         response = client.get(self.path('errata'), **kwargs)
         return _handle_response(response, self._server_config, synchronous)
 
+    def traces(self, synchronous=True, **kwargs):
+        """List services that need restarting on the host
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all content decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+        """
+        kwargs = kwargs.copy()  # shadow the passed-in kwargs
+        kwargs.update(self._server_config.get_client_kwargs())
+        response = client.get(self.path('traces'), **kwargs)
+        return _handle_response(response, self._server_config, synchronous)
+
+    def bulk_traces(self, synchronous=True, **kwargs):
+        """List services that need restarting on the specified set of hosts
+
+        :param synchronous: What should happen if the server returns an HTTP
+            202 (accepted) status code? Wait for the task to complete if
+            ``True``. Immediately return the server's response otherwise.
+        :param kwargs: Arguments to pass to requests.
+        :returns: The server's response, with all content decoded.
+        :raises: ``requests.exceptions.HTTPError`` If the server responds with
+            an HTTP 4XX or 5XX message.
+        """
+        kwargs = kwargs.copy()  # shadow the passed-in kwargs
+        kwargs.update(self._server_config.get_client_kwargs())
+        response = client.post(self.path('bulk/traces'), **kwargs)
+        return _handle_response(response, self._server_config, synchronous)
+
+    def resolve_traces(self, synchronous=True, **kwargs):
+        """Resolve traces the host
```

Comment:
Re: Resolve traces the host

I wonder do you mean: Resolve traces for the host, or Resolve traces on the host ?

Reply:
Oh, sure, there is a missing word. Which one would you suggest as a native speaker, `on` or `for`?

Reply:
@ogajduse `for` seems the safe choice as the code doing the resolving is not running on the host.

Reply:
Thanks, it will be addressed in #758.